### ISSUE TITLE
fuckfuckadblock

### ISSUE THREAD

**Issue by FadeMind** [AdguardTeam/FiltersRegistry#196]
Hi, Please consider adding https://github.com/bogachenko/fuckfuckadblock/blob/master/fuckfuckadblock.txt
list with lots of script blocking lots of ads script - pop-ups and anti-adblock bypass.

P.S. Why **CoinBlockerLists** is not available in LIST Adguard For Windows App? (was included?)

**Comment by ameshkov** [comment]
There're two issues with it:

1. Troubling name, we cannot have outright swearing inside the product, despite our feelings towards FAB:)
2. We do not allow using JS filters in third-party filter lists added here. However, when you add it as a custom filter list, you can explicitly allow JS filters and make it work.

> P.S. Why CoinBlockerLists is not available in LIST Adguard For Windows App? (was included?)

Nope, I am troubled with its license (GPL Affero is quite strict) and not sure if we can add it. Waiting for a lawyer's comment now.

**Comment by FadeMind** [comment]
@ameshkov you can FORK  FFA  and renaming (MIT License), or simply MERGE with Yours CODE BLOB. I added some new reports and script from maintainer works well. See ref commit https://github.com/AdguardTeam/AdguardFilters/issues/33665

Regards

**Comment by ameshkov** [comment]
@FadeMind yeah, the maintainer does a great job indeed, but the point 2 still stands. 

When we add scriptlets support to AG, we might allow them for third-party filter lists, that will solve the issue.

**Comment by bogachenko** [comment]
@ameshkov Я **не** подгоняю, но есть ли примерная дата релиза скриптлетов для программы/расширения сафари. И кстати у вас есть календарь релизов, типа как у тормозиллы `https://wiki.mozilla.org/Release_Management/Calendar` ? Планируете запилить нечто подобное?

Хорошего вам дня!

**Comment by ameshkov** [comment]
@bogachenko привет!

Публичного календаря у нас нет, но вообще релизы у нас раз в 2-3 месяца выходят. Вот где-то месяца через 2 и будут скриптлеты.

> Создадите еще 1 страницу для подобной таблицы? Пжлст. Или это лишний геморрой?

Мы просто не настолько четко планируем, чтобы это публичным делать:(

**Comment by lancelot-moon** [comment]
> We do not allow using JS filters in third-party filter lists added here.
@ameshkov 
In AG Browser Extension, I don't see `trusted` symbol for those third-party filters in the default filters list.

But I see those third-party filters in the default filters list are marked as `trusted` in AG for Windows.
So JS filters in those third-party filters of the default filters list in AG for Windows/Mac will work, right?

**Comment by ameshkov** [comment]
All filters that we re-host on our servers are marked as trusted by default.
However, we remove all dangerous rules from these re-hosted versions.

**Comment by lancelot-moon** [comment]
Are **all** third-party filters in the default filters list re-host in all AG products?
Ex: `EasyList China`, `Adblock Warning Removal List`...etc.
I supposed those third-party filters in the default filters list are subscribed from the official link.
Anyway, thanks for the information.

**Comment by ameshkov** [comment]
@bogachenko вопрос - нас просят (не только тут) добавить твой фильтр в рекомендованные, но мы не можем из-за его названия. Есть ли опция замутить какой-то safe-for-work форк?

**Comment by bogachenko** [comment]
@ameshkov можно легче - https://github.com/PolishFiltersTeam/PolishAnnoyanceFilters.netlify.com/blob/8dfe200e9f57651a4eca88db75ca9a544fc8c49e/data/FilterLists.toml#L269

**Comment by ameshkov** [comment]
@bogachenko да со ссылкой на homepage проблема, название то сделать можно всякое. Ну ладно, может и такой вариант пройдет.

**Comment by bogachenko** [comment]
@ameshkov 
>Ну ладно, может и такой вариант пройдет.
>пройдет

Так, для тупых. Вы типа там голосуете?

**Comment by ameshkov** [comment]
@bogachenko не, юзеры слабонервные могут расстроиться.

**Comment by bogachenko** [comment]
Ну всем не угодишь, если сомневаешься - не добавляй. Пользователи сами найдут, если надо будет.
***
В любом случае создавать safe-for-work форк - не.

> не, юзеры слабонервные могут расстроиться.

 `https://github.com/sitexw/FuckAdBlock/issues/42`

🤣

**Comment by ameshkov** [comment]
Время сейчас такое, триггерятся со всего

**Comment by FadeMind** [comment]
@ameshkov What is big deal with FORK and using scriptlets from FFA DIRECTLY in Custom Named Adguard AntiAdblocks (R) 😉  separelly as standalone optional repository BASED on FFA and merged in period time? Second, Lots of fixes from FFA are not available in Adguard. (For example about:blank freak out some ignored by Adguard Filters Team, resolved in FFA). Some sites sometimes did not work properly, but at end REPORT and FEEDBACK from MAINTAINER is GOLDEN @bogachenko

**Comment by ameshkov** [comment]
@FadeMind well, first of all, I'd like the real maintainer to get praised for the filter list.

What for the fixes, @Alex-302 do you know about these issues?

**Comment by FadeMind** [comment]
@ameshkov take me some time for research and will post proper examples ;)

**Comment by Alex-302** [comment]
> do you know about these issues?

no.

**Comment by FadeMind** [comment]
@ameshkov 
https://github.com/AdguardTeam/AdguardFilters/issues/36056
https://github.com/AdguardTeam/AdguardFilters/issues/33665

**Comment by lancelot-moon** [comment]
fuckfuckadblock causes some websites' comments not to be correctly shown.
People will be unable to leave comments on some websites with it.

**Comment by bogachenko** [comment]
@FadeMind I deleted most of them

**Comment by lancelot-moon** [comment]
@bogachenko 
https://hiraku.tw/2020/02/6044/
With fuckfuckadblock, try commenting and then you'll find that some comment buttons are incorrectly blocked.
You'll be unable to "send" comment.
